timemachine/fe/single_topology.py

```diff
@@ -0,0 +1,783 @@
+from collections.abc import Iterable
+
+import numpy as np
+
+from timemachine.fe import dummy, geometry, topology
+from timemachine.fe.geometry import LocalGeometry
+
+
+def is_planarizing(force_constant, phase, period):
+    return period == 2 and (phase - np.pi) < 0.05 and force_constant > 10.0
+
+
+def identify_bonds_spanned_by_planar_torsions(proper_idxs, proper_params):
+    """
+    Identify bonds that are spanned by planar torsions and returns a dict of bonds
+    and associated torsions that span it.
+    """
+    planar_bonds = dict()
+
+    # collect unique idxs first
+    torsions = dict()
```

Comment:
nit: `torsions = defaultdict(list)`

then you can remove `if idxs not in torsions: ...`

Reply:
willfix